server/expression.js

```diff
@@ -211,38 +212,21 @@ expression.readTFAmat_ = function(buf) {
  * @private
  */
 expression.getTfaProfile_ = function(fileName, geneNames, conditionNames) {
-  var bufTfa = utils.readFileToBuf(fileName);
-  if (bufTfa == null) {
-    console.error('cannot read file', fileName);
-    return null;
-  }
-  var resultTfa = expression.readTFAmat_(/** @type {!Buffer} */(bufTfa));
-
+  var result = expression.readExpression_(fileName, geneNames, conditionNames);
```

Comment:
Please add comments to indicate the reason we are calling readExpression here (Because the TFA profile has the same format as an expression matrix...etc).